ansible/roles/test/files/ptftests/copp_tests.py

```diff
@@ -126,6 +143,15 @@ def copp_test(self, packet, count, send_intf, recv_intf):
         e_c_1 = self.dataplane.get_counters(*recv_intf)
         e_n_0 = self.dataplane.get_nn_counters(*send_intf)
         e_n_1 = self.dataplane.get_nn_counters(*recv_intf)
+
+        if self.needAbsorbCBS:
+            '''
+            Restore the received packets('queue') in the pre-send
+            '''
+            self.dataplane.packet_queues[(recv_intf[0],recv_intf[1])]=queue
+            rcv_pkt_cnt = testutils.count_matched_packets(self, packet, recv_intf[1], recv_intf[0])
```

Comment:
Can you please explain the logic of your change? From my understanding what do you want to do is:
1. Send traffic for 1 second.
2. Flush all received traffic
3. Start the regular test we had before

Reply:
As I didn't know timeout value in count_matched_packets() can be < 1, I needed count the value off-line.
Now, this part code is removed.
I add a **needPreSend** parameter to discriminate PolicyTest from NoPolicyTest, because in my test, the new steps (1 and 2) cause the received counter in step 3 NOT up to 100,000, the difference is several-hundred-packets.